sdk/src/client/node_api/core/routes.rs

```diff
@@ -249,7 +249,7 @@ impl ClientInner {
             .await
     }
 
-    /// Returns the metadata of the block that was included in the ledger for a given TransactionId.
+    /// Returns the metadata of the earliest block containing the tx that get confirmed.
```

Comment:
```suggestion
    /// Returns the metadata of the earliest block containing the tx that got confirmed.
```
Idk, it just reads weird to me in the present tense, maybe @Alexandcoats can tell

Reply:
Both are weird as an american english speaker haha. I would say

```/// Returns the metadata of the earliest block containing the tx that was confirmed.```

Reply:
Used this